docs/source/howto/html/if_task.html

```diff
@@ -61,7 +61,7 @@
me": "result"}, {"name": "result"}], "position": [60, 60], "children": []}, "add2": {"label": "add2", "node_type": "CALCFUNCTION", "inputs": [{"name": "x"}, {"name": "y"}, {"name": "x"}], "properties": {"metadata": null, "metadata.store_provenance": null, "metadata.description": null, "metadata.label": null, "metadata.call_link_label": null, "metadata.disable_cache": null, "x": null, "y": null, "_wait": null}, "outputs": [{"name": "result"}], "position": [90, 90], "children": []}, "if_true": {"label": "if_true", "node_type": "IF", "inputs": [{"name": "conditions"}], "properties": {"_wait": null, "conditions": null, "invert_condition": null}, "outputs": [], "position": [120, 120], "children": ["add2"]}, "multiply1": {"label": "multiply1", "node_type": "CALCFUNCTION", "inputs": [{"name": "x"}, {"name": "y"}, {"name": "x"}], "properties": {"metadata": null, "metadata.store_provenance": null, "metadata.description": null, "metadata.label": null, "metadata.call_link_label": null, "metadata.disable_cache": null, "x": null, "y": null, "_wait": null}, "outputs": [{"name": "result"}], "position": [150, 150], "children": []}, "if_false": {"label": "if_false", "node_type": "IF", "inputs": [{"name": "conditions"}], "properties": {"_wait": null, "conditions": null, "invert_condition": null}, "outputs": [], "position": [180, 180], "children": ["multiply1"]}, "select1": {"label": "select1", "node_type": "NORMAL", "inputs": [{"name": "condition"}, {"name": "true"}, {"name": "false"}, {"name": "true"}, {"name": "false"}, {"name": "condition"}], "properties": {"condition": null, "true": null, "false": null, "_wait": null}, "outputs": [{"name": "result"}], "position": [210, 210], "children": []}, "add3": {"label": "add3", "node_type": "CALCFUNCTION", "inputs": [{"name": "x"}, {"name": "y"}, {"name": "x"}], "properties": {"metadata": null, "metadata.store_provenance": null, "metadata.description": null, "metadata.label": null, "metadata.call_link_label": null, "metadata.disable_cache": null, "x": null, "y": null, "_wait": null}, "outputs": [], "position": [240, 240], "children": []}}, "links": [{"from_socket": "result", "from_node": "add1", "to_socket": "x", "to_node": "add2", "state": false}, {"from_socket": "result", "from_node": "condition1", "to_socket": "conditions", "to_node": "if_true", "state": false}, {"from_socket": "result", "from_node": "add1", "to_socket": "x", "to_node": "multiply1", "state": false}, {"from_socket": "result", "from_node": "condition1", "to_socket": "conditions", "to_node": "if_false", "state": false}, {"from_socket": "result", "from_node": "add2", "to_socket": "true", "to_node": "select1", "state": false}, {"from_socket": "result", "from_node": "multiply1", "to_socket": "false", "to_node": "select1", "state": false}, {"from_socket": "result", "from_node": "condition1", "to_socket": "condition", "to_node": "select1", "state": false}, {"from_socket": "result", "from_node": "select1", "to_socket": "x", "to_node": "add3", "state": false}]}
```

Comment:
You can think `workgraphData = get_short_versioin(wg.to_dict())`, so it contains all the information on the graph and tasks, inputs, outputs, links etc.

Most of the time, only the uuid changes if we use the same workgraph version. However, because I ran it with new WorkGraph version, so the data is also changed.